tsconfig.base.json

```diff
@@ -0,0 +1,41 @@
+{
+  "compilerOptions": {
+    "module": "esnext",
+    "target": "ES2015",
+    "lib": ["es2017", "dom"],
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "skipDefaultLibCheck": true,
+    "sourceMap": true,
+    "strict": true,
+    "strictFunctionTypes": false,
+    "moduleResolution": "node",
+    "jsx": "preserve",
```

Comment:
Why this default?

Reply:
the original proposal is about using babel for transpilation.